numpy_basics/numpy_basics.ipynb

```diff
@@ -0,0 +1,2780 @@
+{
```

Comment:
<p>Variable change in the previous cell: re running will cause error.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/syncdoth/ML_Study_2020/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>